src/lib/core/observe-content/observe-content.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Directive,
+  ElementRef,
+  NgModule,
+  ModuleWithProviders,
+  Output,
+  EventEmitter
+} from '@angular/core';
+
+@Directive({
+  selector: '[cdk-observe-content]'
+})
+export class MdMutationObserver {
+  private observer: MutationObserver;
+
+  @Output('cdk-observe-content') event = new EventEmitter();
+
+  constructor(private elementRef: ElementRef) {}
+
+  ngAfterViewInit() {
+    this.observer = new MutationObserver(mutations => mutations.forEach(() => this.event.emit()));
```

Comment:
Might be a good idea to have a method to clean up the mutation observer, otherwise we'll end up with memory leaks at some point.